Library/Formula/agda.rb

```diff
@@ -20,14 +18,17 @@ class Agda < Formula
   head "https://github.com/agda/agda.git", :branch => "master"
 
   option "without-stdlib", "Don't install the Agda standard library"
-  option "with-malonzo-ffi",
-    "Include the MAlonzo backend's FFI (depends on the standard library)"
+  option "without-malonzo", "Don't install GHC (needed for the MAlonzo backend)"
```

Comment:
This option does not work, and moreover, it _should_ not work, as I describe in #45318.